File: TelegramAPI/connectioninfo.cpp
#include "connectioninfo.hpp"

ConnectionInfo::ConnectionInfo(std::string host, std::string port)
{
    _host = host;
    _port = port;
}

ConnectionInfo::~ConnectionInfo()
{
}

std::string ConnectionInfo::getHost()
{
    return _host;
}
std::string ConnectionInfo::getPort()
{
    return _port;
}

File: src/MTProto/Connection/ConnectionInfo.cs
namespace TelegramApi.MTProto.Connection
{
    public class ConnectionInfo : IConnectionInfo
    {
        public ConnectionInfo(string host, int port)
        {
            Host = host;
            Port = port;
        }

        public string Host { get; set; }
        public int Port { get; set; }
    }
}